docker-compose.yml

```diff
@@ -1,13 +1,20 @@
 # docker configuration for running a development server
 version: "3.6"
+
 services:
   job-server:
     build: .
     environment:
+      - ADMIN_USERS=${ADMIN_USERS}
```

Comment:
Where the names match you can just [pass these through](https://docs.docker.com/compose/environment-variables/#pass-environment-variables-to-containers) using:
```
environment:
  - ADMIN_USERS
  - DATABASE_URL
  ...
```

Reply:
🤯 That is incredibly useful to know, thank you!